packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -375,19 +375,22 @@ export class PipelineService {
   ): any {
     for (const node of pipeline.nodes) {
       const nodeDef = components.find((n: any) => {
-        n.id === node.op;
+        return n.op === node.op;
       });
       const params = nodeDef?.app_data?.properties?.uihints?.parameter_info;
       if (!params) {
         continue;
       }
       for (const param of params) {
-        if (param.data?.format === 'file') {
+        if (
+          param.data?.format === 'file' &&
+          param.custom_control_id === 'StringControl'
+        ) {
           node.app_data.component_parameters[
-            param.parameter_ref
+            param.parameter_ref.substring(6)
```

Comment:
I'm curious what the 6th index is about and my hunch is that this must be skipping over the `elyra_` prefix.  If so, would it make sense to define a constant (e.g., `ELYRA_PREFIX_LEN`) for this value where a comment could then be added.

Reply:
Good idea - will do!